packages/firebase_dynamic_links/lib/src/firebase_dynamic_links.dart

```diff
@@ -104,3 +140,13 @@ class PendingDynamicLinkDataIOS {
   /// declares higher [minimumVersion] than currently installed.
   final String minimumVersion;
 }
+
+class OnLinkErrorException {
```

Comment:
I think we might want to call this class `OnLinkError` or `OnLinkErrorData` since it doesn't extend `Exception`

Reply:
Fixed in the latest commit: https://github.com/flutter/plugins/pull/1687/commits/b5c45c262941f09ed00b90c4076b7fbfd67871dc

Thank you :)